src/comments/comments.service.ts

```diff
@@ -6,28 +6,51 @@ import { Comment } from './entities/comment.entity';
 import { Post } from "../posts/entities/post.entity";
 import mongoose from "mongoose";
 import { Profile } from "../profiles/entities/profile.entity";
+import { Notification } from "../notifications/enitities/notification.entity";
+import { User } from "../user/entity/User";
+import { UserService } from "../user/user.service";
 
 @Injectable()
 export class CommentsService {
   constructor(@Inject('COMMENT_MODEL') private readonly commentModel: Model<Comment>,
               @Inject('POST_MODEL') private readonly postModel: Model<Post>,
-              @Inject('PROFILE_MODEL') private readonly profileModel: Model<Profile>) {}
+              @Inject('PROFILE_MODEL') private readonly profileModel: Model<Profile>,
+              @Inject('NOTIFICATION_MODEL') private readonly  notificationModel: Model<Notification>,
 
+             ) {}
+
+  /*
+  here we will also add it to the notifications list
+   */
   async create(createCommentDto: CreateCommentDto) {
     const post = await this.postModel
       .findOne({ _id: createCommentDto.postId })
       .exec();
+
+    var  a =  post.username;
```

Comment:
I think its not good to rely on the username cause they might not be unique, you can do this instead:
<img width="698" alt="Screenshot 2022-05-23 at 21 01 35" src="https://user-images.githubusercontent.com/71661786/169888502-6bd04ab0-c8ac-433d-b830-5f0bbb42ce3d.png">
find an owner of the post